app/templates/admin.pug

```diff
@@ -48,6 +48,18 @@ block content
 
   if me.isOnlineTeacher() && !me.isAdmin()
     div.text-center Use the search box above to find and administer student accounts.
+  else if me.isParentAdmin() && !me.isAdmin()
```

Comment:
it would make sense to show entry to `Admin` in top nav dropbox when user is parentAdmin too